Sources/PeripheryKit/Frontend/Commands/ScanBehavior.swift

```diff
@@ -84,6 +84,10 @@ class ScanBehavior {
                         "Periphery is a very precise tool, false positives often turn out to be correct after further investigation."
                 )
             }
+
+            if !filteredDeclarations.isEmpty && configuration.failOnWarnings {
+                throw PeripheryKitError.foundIssues(count: filteredDeclarations.count)
+            }
```

Comment:
There's a call to `updateChecker.notifyIfAvailable()` further below, let's move it to above this condition so that people still get upgrade notices when using strict mode.

Reply:
Makes sense, fixed.